Gemfile

```diff
@@ -29,6 +29,11 @@ gem 'activerecord-import'
 
 group :development, :test do
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'guard', '~>2.16.2', require: false
+  gem 'guard-livereload', '~>2.5.2', require: false
+  gem 'guard-minitest', '~>2.4.6', require: false
```

Comment:
@dtt101 I can see we have minitest in this project, but we are not actively using it as far as I can see. Having a quick scan of the documentation for the gem, it seems to be only for minitest. Would we not be better using the `rspec` focused version? And one for `cucumber` too if it exists?

My suspicion is `minitest` is in our Gemfile lock as a result of how the project was created. At creation time with `rails new` that the option to drop test was not specified.

Reply:
Good catch! I will remove the `guard-minitest` gem and leave hot reload of tests for another PR, just to keep this one simple.